4.6.2/WcfBasicIISHost/WcfServiceTest/Web.config

```diff
@@ -29,6 +29,7 @@
 
   <system.web>
     <compilation debug="true" targetFramework="4.0" />
+    <customErrors mode="Off"/>
```

Comment:
Yes, it is for debugging purpose. I left it in the sample so that whoever using the sample may benefit from it as well. However, you have good point of making clear it's intension. What about leaving it there, and add comment?